src/web/handlers/GestureHandler.ts

```diff
@@ -167,8 +167,18 @@ export default abstract class GestureHandler {
       this.currentState === State.ACTIVE ||
       this.currentState === State.BEGAN
     ) {
+      // Here the order of this if statement and moveToState call is important.
+      // At this point we can use currentState as previuos state, because immediately after changing cursor we call moveToState method.
+      // By checking whether previous state was ACTIVE, we can decide if we should reset the cursor or not.
+      if (
+        this.config.activeCursor &&
+        this.config.activeCursor !== 'auto' &&
```

Comment:
I might not be seeing something, but why check if the `activeCursor` is not `auto`?

Reply:
Originally it was introduced to avoid situations where you have two (or more) handlers attached to one view, and one of them has `activeCursor` set to `"auto"`, while other has some custom cursor. In that case the one with `"auto"` would reset cursor.